src/actinia_core/rest/ephemeral_processing_with_export.py

```diff
@@ -294,6 +294,51 @@ def _export_raster(self, raster_name,
 
         return file_name, output_path
 
+    def _export_strds(self, strds_name, format="GTiff"):
+        """Export a specific strds layer with t.rast.export.
+
+        The result is stored in a temporary directory
+        that is located in the temporary grass database.
+
+        Args:
+            strds_name (str): The name of the strds layer
+            format (str): GTiff (only option)
+
+        Returns:
+            tuple: A tuple (file_name, output_path)
+
+        """
+        suffix = ".tar.bzip2"
+        file_name = strds_name.split("@")[0] + suffix
+        output_path = os.path.join(self.temp_file_path, file_name)
+
+        if format != 'GTiff':
+            format = 'GTiff'
+            self.message_logger.info("Only GTiff driver is supported for STRDS export.")
+
+        module_name = "t.rast.export"
+        args = [
+            "input=%s" % strds_name,
+            "format=%s" % format,
+            "output=%s" % output_path,
+            "compression=%s" % "bzip2"
+        ]
+        # optimized for GTiff
+        create_opts = "createopt=BIGTIFF=YES,COMPRESS=LZW,TILED=YES"
```

Comment:
Might it be necessary to use single quotes?
`create_opts = "createopt='BIGTIFF=YES,COMPRESS=LZW,TILED=YES'"`

Reply:
in the actinia output it looks like this:
```
   {
      "executable": "t.rast.export",
      "id": "exporter_strds_lst_test",
      "mapset_size": 731903,
      "parameter": [
        "input=lst_test",
        "format=GTiff",
        "output=/actinia_core/workspace/temp_db/gisdbase_0d884d0430814e97b3417abf9163f91b/.tmp/lst_test.tar.bzip2",
        "compression=bzip2",
        "createopt=BIGTIFF=YES,COMPRESS=LZW,TILED=YES"
      ],
 ```
 
for existing raster export it looks syntactically the same:
```
    {
      "executable": "r.out.gdal",
      "id": "exporter_raster_lst_test_extract_00001",
      "mapset_size": 731909,
      "parameter": [
        "-fmt",
        "input=lst_test_extract_00001",
        "format=GTiff",
        "output=/actinia_core/workspace/temp_db/gisdbase_d[...]3/.tmp/lst_test_extract_00001.tif",
        "overviews=5",
        "createopt=BIGTIFF=YES,COMPRESS=LZW,TILED=YES"
      ],
```
gdalinfo tells me that it has LZW compression `COMPRESSION=LZW`

So I guess it is fine that way?

Reply:
I agree, if it is the same for an existing (working) raster export, it should be fine.